documentation/related_projects/index.markdown

```diff
@@ -0,0 +1,40 @@
+---
+author: davetcoleman
+comments: false
+date: 2016-11-02 20:00:00+00:00
+layout: page
+slug: related_projects
+title: Related Projects
+---
+
+# Related Projects Aligned with MoveIt! Framework
+
+MoveIt! is designed to be easily extensible as separate plugins or projects. Below is a list of related projects that have been aligned with MoveIt! framework, in alphabetical order of project names.
+
+## bio_ik
+MoveIt kinematics_base plugin based on particle optimization & GA. For ease of use and compatibility with existing code, the bio-ik algorithm is encapsulated as a Moveit kinematics plugin. Therefore, bio-ik can be used as a direct replacement of the default Orocos/KDL-based IK solver. Given the name of an end-effector and a 6-DOF target pose, bio-ik will search a valid robot joint configuration that reaches the given target. [More Info](https://github.com/TAMS-Group/bio_ik)
+
+## descartes
+ROS-Industrial Special Project: Cartesian Path Planner. Descartes is a ROS-Industrial project for performing path-planning on under-defined Cartesian trajectories. Descartes uses trajectory points, robot models, and planners to generate an joint-trajectory that complies with the constraints of a given process. [More Info](https://github.com/ros-industrial-consortium/descartes)
+
+## industrial_moveit
+This stack is part of the ROS Industrial program. It currently contains packages that are meant for use with the MoveIt packages. [More Info](https://github.com/ros-industrial/industrial_moveit)
```

Comment:
This package is deprecated and being split into just a ``stomp`` package. I don't think it should be mentioned here